apps/ensindexer/src/plugins/protocol-acceleration/plugin.ts

```diff
@@ -113,11 +119,19 @@ export default createPlugin({
 
       contracts: {
         // a multi-chain Resolver ContractConfig
-        [namespaceContract(pluginName, "Resolver")]: resolverContractConfig(
-          config.namespace,
-          DATASOURCES_WITH_RESOLVERS,
-          config.globalBlockrange,
-        ),
+        [namespaceContract(pluginName, "Resolver")]: {
+          abi: ResolverABI,
+          chain: getDatasourcesWithResolvers(config.namespace).reduce(
```

Comment:
using the shared `getDatasourcesWithResolvers` to generate the Resolver ContractConfig for the protocol acceleration plugin